src/fhir/api.test.ts

```diff
@@ -0,0 +1,33 @@
+import {Patient} from "@smile-cdr/fhirts/dist/FHIR-R4/classes/patient";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import {Fhir} from "fhir";
+import {createBundle} from "./api";
+
+const fhir = new Fhir();
+
+
+describe("FHIR resources", () => {
+  /**
+   * Given that form data has been correctly populated
+   * When a FHIR bundle is created
+   * Then the fhir library should pass validation of the bundle
+   */
+  test('Bundle is valid', () => {
+    const patientForm = {
+      mrn: "969977",
+      firstName: "Donald",
+      lastName: "Duck",
+      dateOfBirth: "2012-03-04",
+      gender: Patient.GenderEnum.Male,
+      familyNumber: "Z409929",
+    };
+
+    const bundle = createBundle(patientForm);
+
+    const output = fhir.validate(bundle);
```

Comment:
Do we perform this validation in the actual application flow? Or is this handled via a separate method somewhere?

Reply:
Ah we don't currently but we should do to save sending requests which are bound to fail